### PR TITLE
Add additional models, migrations, nova resources and basic tests

### PR DESCRIPTION
**Pull request by pdbreen**
Tests are bare minimum - create & persist each model and simple index get API call for each Nova resource.

### CODE REVIEW

File: src/ReviewsServiceProvider.php [hunk 1]
```diff
@@ -1,42 +1,25 @@
...
-                $this->publishes([
-                    $this->package->basePath("/../database/migrations/{$migrationFileName}.php.stub") => database_path('migrations/' . Str::finish($migrationFileName, '.php')),
-                ], "{$this->package->name}-migrations");
-            }
-        }
+        $package
+            ->name('reviews')
+            ->hasConfigFile();
     }
 }
```
Comment (drewroberts) [inline src/ReviewsServiceProvider.php:25]: Should we update every package to use this structure?

Would it be better to refactor to our own function of `->loadMigrations()` that handles the boot method in the chaining style of Spatie's package tools?

File: tests/TestCase.php [hunk 2]
```diff
@@ -1,41 +1,56 @@
 <?php
 
+declare(strict_types=1);
+
```
Comment (drewroberts) [inline tests/TestCase.php:4]: I'll have someone go through all the packages and add this to every php file.

**Review by drewroberts — APPROVED** [review]
Thank you! 📰

File: src/ReviewsServiceProvider.php (hunk 1 above)
Comment (pdbreen) [reply src/ReviewsServiceProvider.php:25]: Yes.  The changes in this file (along with removing `.stub` extension from the migration files themselves) changes this package from publishing migrations to loading them directly from the package (ie, method #2 in the conversation yesterday).

File: tests/TestCase.php (hunk 2 above)
Comment (pdbreen) [reply tests/TestCase.php:4]: Its not so valuable that its worth dedicated effort IMO, its just habit for me to always add it when I find myself in a file without it.

**Pull request merged**